core/src/main/java/io/seata/core/protocol/MergeResultMessage.java

```diff
@@ -116,7 +116,11 @@ public void decode(ByteBuffer byteBuffer) {
     @Override
     public String toString() {
         StringBuilder sb = new StringBuilder("MergeResultMessage ");
-        for (AbstractMessage msg : msgs) { sb.append(msg.toString()).append("\n"); }
+        if (msgs != null) {
```

Comment:
if(msg == null){
return sb.toString
}
for (AbstractMessage msg : msgs) {
                sb.append(msg.toString()).append("\n");
            }
       
        return sb.toString();